generic/tests/mac_change.py

```diff
@@ -113,6 +114,7 @@ def run(test, params, env):
             if os_variant == "winxp":
                 connection_id = pnpdevice_id.split("&")[-1]
                 mode = "devcon"
+            time.sleep(10)
```

Comment:
if used wait_for()
117             print "111111111111111111111111111111111111111111111111"
118             print utils_misc.wait_for(lambda: utils_net.restart_windows_guest_network(session_serial,
119                                                                                        connection_id,
120                                                                                        mode=mode),timeout=100, first=10)
121             print "22222222222222222222222222222222222222222"

165             print utils_misc.wait_for(lambda: utils_net.restart_windows_guest_network(session_serial,
166                                                                                       connection_id,
167                                                                                       mode=mode), timeout=100, first=10)
test log:
2016-12-15 14:51:19,337 output           L0642 DEBUG| 111111111111111111111111111111111111111111111111
2016-12-15 14:51:29,347 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=DISABLED
2016-12-15 14:51:34,523 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:51:35,632 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=ENABLED
2016-12-15 14:51:36,742 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:51:38,851 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=DISABLED
2016-12-15 14:51:40,976 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:51:42,085 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=ENABLED
2016-12-15 14:52:10,617 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:52:12,727 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=DISABLED
2016-12-15 14:52:13,836 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:52:14,945 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=ENABLED
2016-12-15 14:52:39,414 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:52:41,524 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=DISABLED
2016-12-15 14:52:42,633 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:52:43,742 client           L1049 DEBUG| Sending command: netsh interface set interface name="Ethernet" admin=ENABLED
2016-12-15 14:53:09,226 client           L1049 DEBUG| Sending command: echo %errorlevel%
2016-12-15 14:53:11,336 output           L0642 DEBUG| None
2016-12-15 14:53:11,337 output           L0642 DEBUG| 22222222222222222222222222222222222222222

and also have  this issue.
ERROR 2-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.mac_change.ctrl_mac_addr_off -> ShellTimeoutError: Timeout expired while waiting for shell command to complete: 'reg delete  HKLM\\SYSTEM\\CurrentControlSet\\Control\\Class\\{4D36E972-E325-11CE-BFC1-08002BE10318}\\0001 /v NetworkAddress /f'    (output: '')